### PR TITLE
vaunt: add achievements for docs dash event

### PR DESCRIPTION
**Pull request by elewis787**
Adding Vaunt Achievements for the docs dash event.

### CODE REVIEW

**Comment by elewis787** [comment]
@qiancai @rpaik - here is the Vaunt PR to add achievements to this repo.

**Review by rpaik — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: .vaunt/config.yaml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    - action:
+          type: point
+          name: docs_dash_pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
```
Comment (rpaik) [inline .vaunt/config.yaml:23]: merged_at date should now be 2024-02-01

File: .vaunt/config.yaml [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    - action:
+          type: point
+          name: pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
```
Comment (rpaik) [inline .vaunt/config.yaml:73]: merged_at date should now be 2024-02-01

File: .vaunt/config.yaml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    - action:
+          type: point
+          name: bonus_pr
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
```
Comment (rpaik) [inline .vaunt/config.yaml:112]: merged_at date should now be 2024-02-01

File: .vaunt/config.yaml [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+                        & labels in ['contribution','tidb-docs-dash-bonus']"
```
Comment (rpaik) [inline .vaunt/config.yaml:113]: condition should be `{'contribution','tidb-docs-dash-bonus'}`

**Review by Yuiham — APPROVED** [review]

File: .vaunt/config.yaml (hunk 1 above)
Comment (qiancai) [inline .vaunt/config.yaml:23]:
```suggestion
                        & merged = true & merged_at <= 2024-02-01 20:00:00
```

File: .vaunt/config.yaml (hunk 2 above)
Comment (qiancai) [inline .vaunt/config.yaml:73]:
```suggestion
                        & merged = true & merged_at <= 2024-02-01 20:00:00
```

File: .vaunt/config.yaml (hunk 3 above)
Comment (qiancai) [inline .vaunt/config.yaml:112]:
```suggestion
                        & merged = true & merged_at <= 2024-02-01 20:00:00
```

File: .vaunt/config.yaml (hunk 4 above)
Comment (qiancai) [inline .vaunt/config.yaml:113]:
```suggestion
                        & labels in {'contribution','tidb-docs-dash-bonus'}"
```

**Pull request merged**